package.json

```diff
@@ -39,10 +41,12 @@
     "@types/react": "^17.0.39",
     "@types/react-dom": "^17.0.11",
     "babel-loader": "^8.2.3",
+    "chrome-webstore-upload": "0.5.0",
     "clean-webpack-plugin": "^4.0.0",
     "copy-webpack-plugin": "^7.0.0",
     "crx": "^5.0.1",
     "css-loader": "^6.6.0",
+    "execa": "^6.1.0",
```

Comment:
Hey @wxharry, what does this package do?

Reply:
Oops. I think it's for bumping a version and we don't need it anymore. I will remove it.

Reply:
OK, thanks~